docs/docs/quickstart/README.md

```diff
@@ -0,0 +1,113 @@
+# Introduction
+
+Sign up for a [free Pipedream account (no credit card required)](https://pipedream.com/auth/signup) and complete this quickstart guide to learn the basic patterns for workflow development:
+
+- Trigger workflows on HTTP requests, schedules and app events
+- Return a custom response from your workflow on HTTP requests 
+- Use connected accounts in actions and code steps
+- Pass data between code steps and no code actions
+- Use npm packages in Node.js code steps
+- Scaffold an API request in Node.js
+- End workflow execution early
+
+This guide [starts with simple examples](/quickstart/hello-world) to cover basic Pipedream concepts and builds up to a [real-world use case](/quickstart/real-world-example/) to post richly formatted Tweets to Slack. Many examples in this guide build on the same workflow, so we recommend that you complete them in order.
+
+<!--![image-20210518194229746](./images/image-20210518194229746.png)-->
+
+
+
+<!--![image-20210518194229746](./images/image-20210518194229746.png)-->
+
+<!--
+1. [hello world!](/quickstart/hello-world/)
+2. [hello ${name}!](/quickstart/hello-name/)
+3. [Make outbound HTTP requests](/quickstart/make-http-request/) 
+4. [Use npm packages](/quickstart/using-npm-packages/) 
+5. [Insert data into Google Sheets](/quickstart/add-data-to-google-sheets/) 
+6. [End workflow execution early](/quickstart/end-workflow-early/) 
+7. [Use managed auth in code](/quickstart/use-managed-auth-in-code/) 
+8. [Run a workflow on a schedule](/quickstart/run-workflow-on-a-schedule/) 
+9. [Email yourself on new items in an RSS feed](/quickstart/email-yourself/) 
+10. [Real-world Twitter -> Slack](/quickstart/real-world-example/) 
+-->
+
+<p style="text-align:center;">
+<a href="/quickstart/hello-world/"><img src="./get-started.png"></a>
+</p>
+
+*After you complete this guide and understand the basics, explore [advanced features](/quickstart/next-steps/) like state management, concurrency and execution rate controls and more.*
+
+<!--
```

Comment:
Similar to the comment above - do you need to keep these in comments, or can you delete?

Reply:
done